dhis-2/dhis-test-integration/src/test/resources/tracker/tracker_basic_metadata.json

```diff
@@ -1598,6 +1687,118 @@
       "id": "Xl2V8x8R1R5",
       "password": "Test123###...",
       "attributeValues": []
+    },
+    {
+      "dataViewOrganisationUnits": [
+        {
+          "id": "QfUVllTs6cZ"
+        }
+      ],
+      "teiSearchOrganisationUnits": [],
```

Comment:
if it's not too much trouble right now, can we replace it to t**eS**earchOrganisationUnits

Reply:
We should change it but I would do it in another PR so we don't backport it to 2.41.0